### DISCUSSION TITLE
手模贴图无法渲染出来，是需要设置什么属性吗

### DISCUSSION THREAD

**Discussion by flyingflyfly** [galacean/discussion#33] [Engine] [answered]
1、引擎版本：0.9.0-beta.15
2、模型地址：http://next-ubanner.oss-cn-beijing.aliyuncs.com/AE_BODY_MOVIN/TMP/e5af4aeb0bb0e461b979537fe85b0dfd/m1.gltf
3、blender中效果：
![image](https://user-images.githubusercontent.com/15390663/201591674-3995d11c-405f-4272-8c9d-b0e8b8aaf071.png)
4、小程序中效果：
![image](https://user-images.githubusercontent.com/15390663/201591758-7c9f4df6-e902-4b16-ae2e-677a153adf25.png)
5、gltf查看器效果：
![image](https://user-images.githubusercontent.com/15390663/201591851-0460e801-a27b-44d7-b999-928194c36ba1.png)

**Comment by KingCyril** [comment]
![image](https://user-images.githubusercontent.com/85378736/201594699-284537b6-4064-47d0-9b6f-f083d3446827.png)
three editor预览这个模型可以

**Comment by JujieX** [comment, marked as answer]
模型使用了顶点着色
